e2e/isolated/isolated_test.go

```diff
@@ -0,0 +1,247 @@
+				AppName:       appName,
+				EnvName:       envName,
+				Profile:       "default",
+				Prod:          false,
+				VPCImport:     vpcImport,
+				CustomizedEnv: true,
+			})
+		})
+
+		It("env init should succeed for 'private' env", func() {
+			Expect(testEnvInitErr).NotTo(HaveOccurred())
+		})
+		It("env ls should list private env", func() {
+			envListOutput, err := cli.EnvList(appName)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(len(envListOutput.Envs)).To(Equal(1))
+			env := envListOutput.Envs[0]
+			Expect(env.Name).To(Equal(envName))
+			Expect(env.Prod).To(BeFalse())
+			Expect(env.ExecutionRole).NotTo(BeEmpty())
+			Expect(env.ManagerRole).NotTo(BeEmpty())
+		})
+	})
+
+	Context("when creating and deploying a backend service in private subnets", func() {
+		var initErr error
+		BeforeAll(func() {
+			_, initErr = cli.SvcInit(&client.SvcInitRequest{
+				Name:       svcName,
+				SvcType:    "Backend Service",
+				Dockerfile: "./backend/Dockerfile",
+				SvcPort:    "80",
+			})
+		})
+
+		It("should not return an error", func() {
+			Expect(initErr).NotTo(HaveOccurred())
+		})
+		It("svc init should create a svc manifest", func() {
+			Expect("./copilot/backend/manifest.yml").Should(BeAnExistingFile())
+		})
+		It("should write 'http' and private placement to the manifest", func() {
+			f, err := os.OpenFile("./copilot/backend/manifest.yml", os.O_WRONLY|os.O_APPEND, 0644)
+			Expect(err).NotTo(HaveOccurred(), "should be able to open the file to append content")
+			_, err = f.WriteString(`
+http:
+  path: '/'
+network:
+  vpc:
+    placement: 'private'
+`)
+			Expect(err).NotTo(HaveOccurred(), "should be able to write 'private' placement to manifest file")
+			err = f.Close()
+			Expect(err).NotTo(HaveOccurred(), "should have been able to close the manifest file")
+		})
+		It("svc ls should list the svc", func() {
+			svcList, svcListError := cli.SvcList(appName)
+			Expect(svcListError).NotTo(HaveOccurred())
+			Expect(len(svcList.Services)).To(Equal(1))
+			Expect(svcList.Services[0].Name).To(Equal("backend"))
+		})
+	})
+
+	Context("when deploying a svc to 'private' env", func() {
+		var privateEnvDeployErr error
+		BeforeAll(func() {
+			_, privateEnvDeployErr = cli.SvcDeploy(&client.SvcDeployInput{
+				Name:    svcName,
+				EnvName: envName,
+			})
+		})
+
+		It("svc deploy should succeed", func() {
+			Expect(privateEnvDeployErr).NotTo(HaveOccurred())
+		})
+	})
+
+	Context("when running svc show to retrieve the service configuration, resources, and endpoint, then querying the service", func() {
+		var (
+			svc          *client.SvcShowOutput
+			svcShowError error
+		)
+		BeforeAll(func() {
+			svc, svcShowError = cli.SvcShow(&client.SvcShowRequest{
+				Name:      svcName,
+				AppName:   appName,
+				Resources: true,
+			})
+		})
+
+		It("should not return an error", func() {
+			Expect(svcShowError).NotTo(HaveOccurred())
+		})
+
+		It("should return correct configuration", func() {
```

Comment:
can we add an additional test case for `svc status` to ensure the task successfully spun up?
similar to here:
https://github.com/aws/copilot-cli/blob/1caa364fd59f1bab4e0bc6a09d1d271fff24f5ab/e2e/multi-svc-app/multi_svc_app_test.go#L254